pkg/executables/testdata/kind_config_registry_mirror_insecure.yaml

```diff
@@ -0,0 +1,23 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1beta2
+    kind: ClusterConfiguration
+    dns:
+      type: CoreDNS
+      imageRepository: public.ecr.aws/eks-distro/coredns
+      imageTag: v1.8.0-eks-1-19-2
```

Comment:
We should probably use 1-21 artifacts here and also change other tests to reference 1.21 artifacts (though not in this PR).

Reply:
We can update all of those in a follow up PR